cedar-policy-core/cedar-policy-macros/tests/pass.rs

```diff
@@ -0,0 +1,17 @@
+use cedar_policy_core::ast::{Eid, EntityUID};
+use cedar_policy_macros::euid;
+
+fn main() {
+    let e: EntityUID = euid!(Foo, "bar");
```

Comment:
If this returns `core::EntityUID` how do consumers of `cedar-policy` use it?

Reply:
I think this macro can only be used by cedar-policy because of circular dependencies. That is, if we use `core::EntityUID` to parse the input, we cannot export it to core since that would form a mutual reference. I probably miss something (e.g., crate vs. package) given my limited knowledge of Rust. I'm open to suggestions.